<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FenPrincipale</class>
 <widget class="QWidget" name="FenPrincipale">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>910</width>
    <height>703</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion de production - Edition Pro 2011</string>
  </property>
  <widget class="QLabel" name="lBackImage">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>721</width>
     <height>211</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="ressources.qrc">:/images/ressources/ihm.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lCaptDim">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="ressources.qrc">:/images/ressources/ok.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="lCaptPresC">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>70</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="ressources.qrc">:/images/ressources/ok.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="lCaptImpr">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>70</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="ressources.qrc">:/images/ressources/ok.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="lEtatFile">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>70</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="ressources.qrc">:/images/ressources/ok.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="lCaptPresP">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="ressources.qrc">:/images/ressources/ok.png</pixmap>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdFile">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>160</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="numDigits">
    <number>2</number>
   </property>
  </widget>
  <widget class="QTextEdit" name="tLog">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>290</y>
     <width>701</width>
     <height>121</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>640</y>
     <width>491</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="boutonLayout">
    <item>
     <widget class="QPushButton" name="bRunProd">
      <property name="text">
       <string>Lancer production</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="bNewCom">
      <property name="text">
       <string>Nouvelle commande</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="bExpCom">
      <property name="text">
       <string>Exp√©dier commande</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>420</y>
     <width>701</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="tComm"/>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections/>
</ui>
